### PR TITLE
New package: DWANGO.OpenToonz version 1.7.1

### LINKED ISSUE #124865: [Package Request]: OpenToonz
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/opentoonz/opentoonz/releases
Publisher: [DWANGO](http://dwango.co.jp/english/)
Package Name: opentoonz
Description: OpenToonz is a 2D animation software 
Package Version: V1.7.1
Installer URL: https://github.com/opentoonz/opentoonz/releases/download/v1.7.1/OpenToonzSetup.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve #124865

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/125044)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Launching executable file: C:\Program Files\OpenToonz\lzocompress.exe
Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)

File: manifests/d/DWANGO/OpenToonz/1.7.1/DWANGO.OpenToonz.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: DWANGO.OpenToonz
+PackageVersion: 1.7.1
+InstallerType: inno
+ProductCode: '{DF519282-600D-4E03-9190-6046329B1CB4}_is1'
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/d/DWANGO/OpenToonz/1.7.1/DWANGO.OpenToonz.installer.yaml:9]:
```suggestion
- Architecture: x64
```

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

```
OpenToonz 1.7.1  (Build Mar 14 
> 2023)

Report Date: 
> 2023-11-14 10:33:05
Crash Reason: EXCEPTION_ACCESS_VIOLATION

Build ABI: x86_64-little_endian-llp64
Operating System: Windows 10 Version 2009
OS Kernel: 10.0.19044
CPU Threads: 8


Crash File: C:\Users\User\AppData\Local\OpenToonz\OpenToonz\crash\Crash-
> 20231114-103305.log
Mini Dump File: C:\Users\User\AppData\Local\OpenToonz\OpenToonz\crash\Crash-
> 20231114-103305.dmp

==== Modules ====
C:\Program Files\OpenToonz\OpenToonz.exe
C:\Windows\SYSTEM32\ntdll.dll
C:\Windows\System32\KERNEL32.DLL
C:\Windows\System32\KERNELBASE.dll
C:\Windows\System32\USER32.dll
C:\Windows\System32\win32u.dll
C:\Program Files\OpenToonz\Qt5Xml.dll
C:\Program Files\OpenToonz\Qt5PrintSupport.dll
C:\Windows\System32\GDI32.dll
C:\Windows\SYSTEM32\OPENGL32.dll
C:\Windows\System32\gdi32full.dll
C:\Windows\System32\msvcp_win.dll
C:\Windows\System32\ucrtbase.dll
C:\Windows\System32\msvcrt.dll
C:\Windows\System32\SHELL32.dll
C:\Windows\System32\ADVAPI32.dll
C:\Windows\System32\COMDLG32.dll
C:\Program Files\OpenToonz\glut64.dll
C:\Windows\System32\combase.dll
C:\Windows\System32\ole32.dll
C:\Windows\System32\RPCRT4.dll
C:\Program Files\OpenToonz\Qt5SerialPort.dll
C:\Windows\System32\OLEAUT32.dll
C:\Windows\System32\shcore.dll
C:\Windows\System32\sechost.dll
C:\Windows\System32\SETUPAPI.dll
C:\Windows\System32\SHLWAPI.dll
C:\Program Files\OpenToonz\EDSDK.dll
C:\Windows\System32\cfgmgr32.dll
C:\Windows\System32\bcrypt.dll
C:\Windows\System32\WS2_32.dll
C:\Program Files\OpenToonz\colorfx.dll
C:\Program Files\OpenToonz\image.dll
C:\Program Files\OpenToonz\tnztools.dll
C:\Program Files\OpenToonz\tnzstdfx.dll
C:\Program Files\OpenToonz\Qt5Widgets.dll
C:\Program Files\OpenToonz\tfarm.dll
C:\Program Files\OpenToonz\Qt5Gui.dll
C:\Program Files\OpenToonz\Qt5Core.dll
C:\Program Files\OpenToonz\toonzqt.dll
C:\Program Files\OpenToonz\sound.dll
C:\Program Files\OpenToonz\opencv_world451.dll
C:\Program Files\OpenToonz\toonzlib.dll
C:\Program Files\OpenToonz\Qt5Script.dll
C:\Program Files\OpenToonz\tnzext.dll
C:\Program Files\OpenToonz\tnzbase.dll
C:\Program Files\OpenToonz\tnzcore.dll
C:\Program Files\OpenToonz\Qt5OpenGL.dll
C:\Program Files\OpenToonz\Qt5Multimedia.dll
C:\Program Files\OpenToonz\Qt5Network.dll
C:\Windows\SYSTEM32\GLU32.dll
C:\Windows\SYSTEM32\MPR.dll
C:\Windows\System32\CRYPT32.dll
C:\Windows\SYSTEM32\dbghelp.dll
C:\Program Files\OpenToonz\MSVCP140.dll
C:\Program Files\OpenToonz\MSVCP140_1.dll
C:\Program Files\OpenToonz\VCRUNTIME140.dll
C:\Windows\SYSTEM32\WINSPOOL.DRV
C:\Windows\SYSTEM32\WINMM.dll
C:\Program Files\OpenToonz\VCRUNTIME140_1.dll
C:\Windows\SYSTEM32\MFPlat.DLL
C:\Windows\SYSTEM32\MF.dll
C:\Windows\WinSxS\amd64_microsoft.windows.common-controls_6595b64144ccf1df_5.82.19041.1110_none_792d1c772443f647\COMCTL32.dll
C:\Windows\SYSTEM32\MSVFW32.dll
C:\Windows\SYSTEM32\VERSION.dll
C:\Windows\SYSTEM32\AVIFIL32.dll
C:\Program Files\OpenToonz\libmypaint-1-4-0.dll
C:\Program Files\OpenToonz\glew32.dll
C:\Windows\SYSTEM32\d3d11.dll
C:\Windows\SYSTEM32\dxgi.dll
C:\Windows\SYSTEM32\USERENV.dll
C:\Windows\SYSTEM32\NETAPI32.dll
C:\Program Files\OpenToonz\Qt5Svg.dll
C:\Windows\SYSTEM32\dwmapi.dll
C:\Windows\SYSTEM32\UxTheme.dll
C:\Program Files\OpenToonz\CONCRT140.dll
C:\Windows\SYSTEM32\IPHLPAPI.DLL
C:\Windows\SYSTEM32\DNSAPI.dll
C:\Windows\SYSTEM32\MSACM32.dll
C:\Program Files\OpenToonz\libjson-c-2.dll
C:\Program Files\OpenToonz\libintl-8.dll
C:\Program Files\OpenToonz\libiconv-2.dll
C:\Windows\SYSTEM32\winmmbase.dll
C:\Windows\SYSTEM32\NETUTILS.DLL
C:\Windows\SYSTEM32\dbgcore.DLL
C:\Windows\SYSTEM32\SRVCLI.DLL
C:\Windows\SYSTEM32\MFCORE.DLL
C:\Windows\SYSTEM32\powrprof.dll
C:\Windows\SYSTEM32\ksuser.dll
C:\Windows\SYSTEM32\kernel.appcore.dll
C:\Windows\SYSTEM32\CRYPTBASE.DLL
C:\Windows\System32\IMM32.DLL
C:\Windows\System32\NSI.dll
C:\Windows\System32\bcryptPrimitives.dll
C:\Windows\SYSTEM32\RTWorkQ.DLL
C:\Windows\SYSTEM32\UMPDC.dll
C:\Program Files\OpenToonz\platforms\qwindows.dll
C:\Windows\SYSTEM32\WTSAPI32.dll
C:\Windows\SYSTEM32\windows.storage.dll
C:\Windows\SYSTEM32\Wldp.dll
C:\Windows\SYSTEM32\profapi.dll
C:\Program Files\OpenToonz\styles\qwindowsvistastyle.dll
C:\Program Files\OpenToonz\iconengines\qsvgicon.dll
C:\Windows\system32\d3d9.dll
C:\Windows\System32\MSCTF.dll
C:\Windows\SYSTEM32\textinputframework.dll
C:\Windows\System32\CoreUIComponents.dll
C:\Windows\SYSTEM32\ntmarta.dll
C:\Windows\System32\CoreMessaging.dll
C:\Windows\SYSTEM32\wintypes.dll
C:\Windows\System32\clbcatq.dll
C:\Windows\System32\AppXDeploymentClient.dll
C:\Windows\system32\dataexchange.dll
C:\Windows\system32\dcomp.dll
C:\Windows\system32\twinapi.appcore.dll
C:\Windows\system32\dwrite.dll
==== End ====

==== Backtrace ====
  1> RtlDeleteAce [0x7ffa01388a3c] <ntdll.dll>
  2> RtlRaiseException [0x7ffa01361276] <ntdll.dll>
  3> KiUserExceptionDispatcher [0x7ffa013b0bfe] <ntdll.dll>
  4> TColorStyle::getRasterStyleFx [0x7ff9c22f5f49] <tnzcore.dll>
  5> TOStream::openChild [0x7ff9c249f050] <tnzcore.dll>
  6> TProject::save [0x7ff9c35947f9] <toonzlib.dll>
  7> TProjectManager::createSandboxIfNeeded [0x7ff9c358e0b0] <toonzlib.dll>
  8> TSmartPointerT<TXshSoundLevel>::~TSmartPointerT<TXshSoundLevel> [0x7ff68a0e0e53] <OpenToonz.exe>
  9> TSmartPointerT<TXshSoundLevel>::~TSmartPointerT<TXshSoundLevel> [0x7ff68a0e2d41] <OpenToonz.exe>
 10> TSmartPointerT<TXshSoundTextLevel>::~TSmartPointerT<TXshSoundTextLevel> [0x7ff68a50eab7] <OpenToonz.exe>
 11> TSmartPointerT<TXshSoundTextLevel>::~TSmartPointerT<TXshSoundTextLevel> [0x7ff68a50e092] <OpenToonz.exe>
 12> BaseThreadInitThunk [0x7ffa00197034] <KERNEL32.DLL>
 13> RtlUserThreadStart [0x7ffa01362651] <ntdll.dll>
==== End ====


```

**Comment by Dragon1573** [comment]
@stephengillie - According to https://github.com/opentoonz/opentoonz/issues/5171#issuecomment-1812395335, a collaborator instruct me to try the latest nightly build version.

Shall I create a new pull request for that nightly version?

- [2023-10-31 06:05:24 UTC](https://github.com/opentoonz/opentoonz_nightlies/releases/download/nightly_b7470844aec8f730d1e4bc5a841d86bd53ddee6c/OpenToonzSetup.exe)

**Comment by stephengillie** [comment]
@Dragon1573,

Yes, please do.

**Comment by Dragon1573** [comment]
This pull request have the same reason of https://github.com/microsoft/winget-pkgs/pull/126821#issuecomment-1816727821. I think it's acceptable.

**Comment by Dragon1573** [comment]
Nightly version is successfully merged. Close this as rejected.

**Pull request closed without merging**